tests/test_examples.py

```diff
@@ -234,6 +234,7 @@ def to_test(
             "codellama/CodeLlama-13b-Instruct-hf",
             "MIT/ast-finetuned-speech-commands-v2",
             "meta-llama/LlamaGuard-7b",
+            "huggyllama/llama-7b",
```

Comment:
Can we perform the test with a more recent version of Llama? This one is Llama v1.

Reply:
We can add the test on any version of Llama. 
We added v1 as for llama2 and 3 we need hugging face token authorization to access the model. If not, it will fail.
Is there a way to pass the token? (or) what do you suggest?

Reply:
For instance I use this script for the DeepSpeed CI: https://github.com/huggingface/optimum-habana/blob/main/tests/ci/slow_tests_deepspeed.sh
It takes a token as an argument to log in. But not sure you do it like that.

Reply:
We added this test to check the working of CP feature.
For llama 2 and 3.1, We can add one more for long sequence length.
Can I add that later along with restructuring?

Reply:
Yes sure, but I would like to do that before release

Reply:
@regisss Do we have any existing tests for Llama2 or Llama3.1 as part of OH. 
How do we pass token there? Can you point out the same?

Reply:
If not, I think we should add token similar to https://github.com/huggingface/optimum-habana/blob/main/tests/ci/slow_tests_deepspeed.sh for https://github.com/huggingface/optimum-habana/blob/main/tests/ci/slow_tests_8x.sh as well

I will update the test to Llama3.1. Can you add token to the slow_tests_8x.sh?
Will add it along with restructuring in separate commit.

If you agree for that. Can you get this merged?

Reply:
Okay, let's do that :+1:

Reply:
@bhargaveede I just pushed a commit to add HF login to the 8x slow tests: https://github.com/huggingface/optimum-habana/commit/9f9b41e6b718f9972b8c349368bb88fa0584bd4c